cards/condyle-fracture-lateral/condyle-fracture-lateral

```diff
@@ -0,0 +1,60 @@
+---
+
+title: Elbow Injuries - Condyle Fracture Lateral
+
+authors:
+   - Melody Glenn, MD
+   - Andrew Little, DO
+   - Mary Haas, MD
+Created: 2016/12/12
+updates: null
+Categories:
+   - Peer Reviewed
+   - Orthopedics
+
+---
+
+# Elbow Injuries: Condyle Fracture Lateral
+
+![](Image 1)
```

Comment:
Once the image file is renamed to `image-1.png` this markdown should be:

```
![](image-1.png)
```
It needs to point to the exact file name of the corresponding image file.